charts/ccsm-helm/test/zeebe-gateway/service_test.go

```diff
@@ -66,3 +66,21 @@ func (s *serviceTest) TestContainerSetGlobalAnnotations() {
 	// then
 	s.Require().Equal("bar", service.ObjectMeta.Annotations["foo"])
 }
+
+func (s *serviceTest) TestContainerServiceAnnotations() {
+	// given
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"annotations.foo": "bar",
```

Comment:
This needs to be changed, otherwise the test will fail :)
```suggestion
			"zeebe-cluster.service.annotations.foo": "bar",
```